### PR TITLE
libexosip 4.1.0 (also added openssl dependency)

### PR DESCRIPTION
**Pull request by L2G**

### CODE REVIEW

File: Library/Formula/libexosip.rb [hunk 1]
```diff
@@ -1,16 +1,18 @@
...
-  depends_on 'pkg-config' => :build
-  depends_on 'libosip'
+  depends_on "pkg-config" => :build
+  depends_on "libosip"
+  depends_on "openssl"
 
   def install
+    # http://growingshoot.blogspot.com/2013_02_01_archive.html
+    ENV.append "LDFLAGS", "-framework CoreFoundation -framework CoreServices "\
+                          "-framework Security"
```
Comment (jacknagel) [inline Library/Formula/libexosip.rb]: This seems like an upstream bug, are they aware?

Comment (L2G) [inline Library/Formula/libexosip.rb]: Not sure. I am looking for a place to report bugs, but I'm not having luck so far. Here's the home page: https://www.antisip.com/eXosip2-toolkit

Comment (L2G) [inline Library/Formula/libexosip.rb]: https://savannah.nongnu.org/support/?group=exosip

Looks like no. I'll try to submit something.

**Comment by L2G** [comment]
The need for extra flags in LDFLAGS has been reported upstream as a bug, [Savannah ticket #45079](https://savannah.nongnu.org/bugs/index.php?45079).

**Comment by DomT4** [comment]
Can you tack the new upstream bug report into the formula above the appropriate lines? Will make it easier to keep track of as it progresses.

**Comment by L2G** [comment]
Done. I'm holding off on pushing this change so as not to waste Jenkins's time.

File: Library/Formula/libexosip.rb [hunk 2]
```diff
@@ -1,16 +1,22 @@
...
-  depends_on 'pkg-config' => :build
-  depends_on 'libosip'
+  depends_on "pkg-config" => :build
+  depends_on "libosip"
+  depends_on "openssl"
 
   def install
+    # Extra linker flags are needed to build this on Mac OS X. See:
+    # http://growingshoot.blogspot.com/2013/02/manually-install-osip-and-exosip-as.html
+    #
```
Comment (DomT4) [inline Library/Formula/libexosip.rb]: You're probably fine to remove the empty comment lines. It's not a huge thing though, I'm just being anal about compact formulae :smile_cat:.

Comment (jacknagel) [inline Library/Formula/libexosip.rb]: Yes, please remove the blank comment lines.

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/72e638dac8c20bd6345e5ad1f9999ba692b020f2. Thanks for your contribution to Homebrew, we appreciate it! :smiley_cat:. If you can add tests in future when formulae don't have them already, that'd be awesome - Feel free to ping if you ever want help with that sort of thing.

**Pull request closed without merging**